cli_reference/get_started_cli.adoc

```diff
@@ -93,9 +115,15 @@ users:
 ----
 ====
 
-Configuration files can be used for example to configure OpenShift to support the setup of multiple servers, namespaces, and users so that you can switch easily between them. Multiple configuration files are allowed; they are loaded at runtime and merged together along with override options specified from the command line. 
+Configuration files can be used for example to configure OpenShift to support
+the setup of multiple servers, namespaces, and users so that you can switch
+easily between them. Multiple configuration files are allowed; they are loaded
+at runtime and merged together along with override options specified from the
+command line.
 
```

Comment:
[Suggestion]
Configuration files allow you to set up multiple users, servers, and namespaces to use with an OpenShift environment. You can use multiple configuration files that are loaded at runtime and combined with the override options specified from the CLI.